src/Templates/ModelTemplate.cshtml

```diff
@@ -414,8 +414,11 @@ if (modelType is DictionaryTypeGo dictionaryType && dictionaryType.SupportsAddit
         }
 
         // Creates a new instance of the @Model.Name type.
-        func New@(Model.Name) (getNextPage @(modelPageType.NextPageFunctionSig)) @Model.Name {
-            return @(Model.Name){@(modelPageType.FnFieldName): getNextPage}
+        func New@(Model.Name) (cur @(modelPageType.ContentType.Name), getNextPage @(modelPageType.NextPageFunctionSig)) @Model.Name {
+        return @(Model.Name){
+        @(modelPageType.FnFieldName): getNextPage,
+        @(modelPageType.ResultFieldName): cur,
+        }
```

Comment:
Visual studio changes the format of the template every time I save the file... The only actual change in this file is here

Reply:
Yes the auto-formatting drives me nuts.